src/common/types/metadata.js

```diff
@@ -1,8 +1,123 @@
-// @flow strict
```

Comment:
Any way we can retain the strict checks?

Reply:
I can look and see. I didn't know the code well enough to understand the proper type, but let me try again.

Reply:
Added this back, with a $FlowFixMe on the one line that was problematic.